ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Ledger/Ledger.hs

```diff
@@ -497,11 +498,12 @@ decodeShelleyLedgerState = decodeVersion [
       (serialisationFormatVersion2, Decode decodeShelleyLedgerState2)
     ]
   where
+    forgetAnnotator d = (\x -> runAnnotator x (Full mempty)) <$> d
     decodeShelleyLedgerState2 :: Decoder s' (LedgerState (ShelleyBlock era))
     decodeShelleyLedgerState2 = do
-      enforceSize "LedgerState ShelleyBlock" 2
+      enforceSize "LedgerState ShelleyBlock" 3
       shelleyLedgerTip        <- decodeWithOrigin decodeShelleyTip
-      shelleyLedgerState      <- fromCBOR
+      shelleyLedgerState      <- forgetAnnotator fromCBOR
```

Comment:
I think this is the wrong thing to do here. This *would* explicitly break with Alonzo. The cbor-in-cbor solution is at least future proof in this case.